include/nrf_rpc/nrf_rpc_ipc.h

```diff
@@ -31,6 +31,14 @@ extern "C" {
  */
 extern const struct nrf_rpc_tr_api nrf_rpc_ipc_service_api;
 
+/** @brief Endpoint state */
+enum nrf_rpc_ipc_state {
+	NRF_RPC_IPC_STATE_UNINITIALIZED, /**< Endpoint is uninitialized */
```

Comment:
```suggestion
        /** Endpoint is uninitialized */
	NRF_RPC_IPC_STATE_UNINITIALIZED, /**
```

Minor comment about documentation style. It is different now than anywhere else

Reply:
This would be moved to *.c file and the comments would be changed to non-doxygen ones.

Reply:
Done